README.md

```diff
@@ -13,6 +13,34 @@ The __filter__ package is generally provided as a folder with source (`.java`) a
 
 A sample set of images are provided in an __"images"__ directory.
 
+## Creating an executable jar file
+If there is no automated build environment set up, the jar file can be created using "jar."
+1. Open up the CLI and change the directory to the projects root directory.
+2. Execute the command "javac Run.java" to compile the Run.java file.
+3. Execute the command "javac \*.java" in the package directories to compile/recompile the java files.
+4. In the root directory execute the command "echo Main-Class: Run > MANIFEST.MF" to create a manifest file you will use later.
+5. Create the jar using the command "jar cfm (DESIRED_FILENAME.jar) MANIFEST.MF \*.class sound/\*.class imagelab/\*.class filters/\*.class"
+6. The new jar file will be named whatever you set it to and it will be located in the projects root directory.
+
+More info on the java commands can be found [here](https://docs.oracle.com/javase/tutorial/deployment/jar/build.html).
```

Comment:
This seems to be a link to information about creating a JAR file, not a link to info about "java commands" (which might be something more like [JDK Tool Specifications](https://docs.oracle.com/en/java/javase/15/docs/specs/man/index.html)).